randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -633,7 +633,7 @@ Extra - room_id: [16]
   > Door to X-B.O.X. Garage (Upper)
       Screw Attack and After Boss X-B.O.X. Defeated
   > Arena
-      Trivial
+      Can Break Single Bomb Blocks
```

Comment:
I may misremember, but didn't anty want to rethink on what this block should become?

Reply:
I mentioned I prefer it being a shot block for future-proofing (I forget if I only imagined it or actually did see a vid of the pb obstruction being ice-clipped through). I forget if there'd been any response.